dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/trackedentity/TrackedEntityOperationParamsMapper.java

```diff
@@ -360,6 +361,12 @@ private int getMaxTeLimit(TrackedEntityQueryParams params) {
   }
 
   private boolean isLocalSearch(TrackedEntityQueryParams params, UserDetails user) {
+    // If the organization unit selection mode is set to CAPTURE,
+    // the search scope becomes irrelevant, and we always treat it as local.
```

Comment:
I don't think the comment "search scope becomes irrelevant" is corret. 

Local Search means, searching in Capture Scope.